src/Menus.cpp

```diff
@@ -103,46 +103,50 @@ void MenuManager::UpdatePrefs()
    mStopIfWasPaused = true;  // not configurable for now, but could be later.
 }
 
-void MenuVisitor::BeginGroup( Registry::GroupItem &item, const Path &path )
+void MenuVisitor::BeginGroup( Registry::GroupItemBase &item, const Path &path )
```

Comment:
It was confusing to find the declaration of this guy not in Menus.h but CommandManager.h

Reply:
Yes:  Menus and CommandManager are tangled up badly, and there is another big pull request in progress to sort that out.